csrankings-b.csv

```diff
@@ -53,6 +53,7 @@ Balbir Barn,Middlesex University,http://www.cs.mdx.ac.uk/people/balbir-barn,2Sw7
 Balbir S. Barn,Middlesex University,http://www.cs.mdx.ac.uk/people/balbir-barn,2Sw7gb4AAAAJ
 Balwinder Sodhi,IIT Ropar,http://www.iitrpr.ac.in/sodhi,NOSCHOLARPAGE
 Bamshad Mobasher,DePaul University,http://facweb.cs.depaul.edu/mobasher,VxhX6sAAAAAJ
+Banafsheh Rekabdar,Portland State University,https://sites.google.com/site/banafsheh1rekabdar/,hjwSKrIAAAAJ

```

Comment:
Home page is a dead link.

Reply:
Link seems to have gone dead recently. I have updated it to a different profile page.